src/pages/marketplace/index.js

```diff
@@ -26,6 +27,18 @@ const Marketplace = ({
       </Head>
       <Main customRouting={props.navigationCustom}>
         <AppBar url={router.asPath} />
+        <HeroWithPrimaryBackgroundAndDesktopScreenshot
+          title={props.title}
+          description={
+            <div dangerouslySetInnerHTML={{ __html: props.description }} />
+          }
+          screenshot={props.screenshot?.data[0]?.url}
+          //making an array of object > url , image
+          logos={props.features_logos.data.map((x) => ({
```

Comment:
This will need a default state (empty array) if the content is not there to prevent the component from breaking

Reply:
Hi Randy,

In the PR I did earlier, I already set the component **HeroWithPrimaryBackgroundAndDesktopScreenshot**, with an empty array on logos prop
![image](https://user-images.githubusercontent.com/44116036/170067659-42ddfa92-5043-49b9-80cc-8e6071ba13fe.png)

This is how the UI looks like without a logos
![image](https://user-images.githubusercontent.com/44116036/170067782-bc5668c9-de6f-4705-b147-9950305a6bee.png)